battenberg/temporary_worktree.py

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
The changes in this file are just a copy operation to make it easier to test. This class used to live in `core.py` and moving it out to isolate it here.